sources/europe/hr/osm-hr-zagreb-2018.geojson

```diff
@@ -0,0 +1,1288 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "osm-hr-zagreb-2018",
+        "name": "osm-hr: Zagreb 2018 Aerial imagery",
+        "icon": "https://osmlab.github.io/editor-layer-index/sources/europe/hr/osm-hr.png",
+        "start_date": "2018",
+        "end_date": "2018",
+        "url": "https://tms.osm-hr.org/zagreb-2018/{zoom}/{x}/{-y}.png",
+        "permission_osm": "explicit",
+        "license_url": "https://narodne-novine.nn.hr/clanci/sluzbeni/2013_02_25_403.html",
```

Comment:
Is this akin to the data being the the public domain?

Reply:
It's the same license as https://github.com/osmlab/editor-layer-index/pull/663 and https://github.com/osmlab/editor-layer-index/pull/643.

Reply:
Oh yes we discussed this in #643, thanks. http://data.gov.hr/open-licence-republic-croatia seems clear to me, except it does require attribution for use so I'd suggest adding the `attribution` field https://github.com/osmlab/editor-layer-index/blob/8838470547becaf52c9d387585813fb66912bb16/schema.json#L145-L159 to all these layers under that license.

Reply:
I added attribution to https://wiki.openstreetmap.org/wiki/Contributors#Croatia and they said it's ok. I can also add attribution field to geojson if you think it's needed.
Also remake of https://wiki.openstreetmap.org/wiki/WikiProject_Croatia/Podloge with proper up to date info is planed

Reply:
I don't think it hurts to also add the attribution here per the schema.

Reply:
Done